cmd/microcloud/remove.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"context"
+
+	"github.com/canonical/microcluster/microcluster"
+	"github.com/spf13/cobra"
+
+	cloudClient "github.com/canonical/microcloud/microcloud/client"
+)
+
+type cmdRemove struct {
+	common *CmdControl
+
+	flagForce bool
+}
+
+func (c *cmdRemove) Command() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "remove <name>",
+		Aliases: []string{"rm"},
+		Short:   "Remove the specified system from all MicroCloud services",
```

Comment:
Should this say "member" instead of "system"?

The other commands like "add" and "cluster" also talk about members not systems.
Also the description of the force flag below says member.